ec2.md

```diff
@@ -34,6 +36,16 @@ hypervisor: ec2
       nfs_server: none
       consoleport: 443
 
+## using role ###
+
+###No fog file needed###
```

Comment:
This doesn't render quite correctly; I think it needs a space between the text and the `###` to render.